google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/PublisherSnippets.java

```diff
@@ -90,8 +90,8 @@ public Publisher getPublisherWithCustomBatchSettings(ProjectTopicName topicName)
       throws Exception {
     // [START pubsub_publisher_batch_settings]
     // Batch settings control how the publisher batches messages
-    long requestBytesThreshold = 5000L; // default : 1kb
-    long messageCountBatchSize = 10L; // default : 100
+    long requestBytesThreshold = 5000L; // default : 1b
+    long messageCountBatchSize = 10L; // default : 1
```

Comment:
can we "qualify" this by saying 1 message?

Reply:
sg!